File: check.pl
# remove separators before checking for other invalid characters
# usage: cat file perl check.pl | ruby check.rb

my $SEP = "\263";
$_ = join('', <STDIN>);
s/\o{347}/c/g;
s/\o{222}/'/g;
print join('<<<<gs>>>>',split($SEP, $_));

File: check.rb
# see if ruby will complain reading input
# usage: cat file | perl check.rb | ruby check.rb

Encoding.default_external = Encoding::UTF_8
i = 0
while x = $stdin.gets
  begin
    i += 1
    x.scan(/e/)
  rescue Exception => e
    puts "#{i} #{x}"
  end
end